lib/adequate_crypto_address/altcoin.rb

```diff
@@ -56,5 +56,10 @@ def valid_address_checksum?
     def checksum(bytes)
       Digest::SHA256.hexdigest(Digest::SHA256.digest([bytes].pack('H*')))[0...8]
     end
+    
+    def litecoin_valid?
+      return unless type == 'Litecoin'
+      valid_prefix?
+    end
```

Comment:
This should be handled by the `Litecoin` class. What is `Altcoin` being considered here?

Reply:
oo yeah, I wasn't sure how to handle adding an additional prefix check for `ltc` that Altcoin wasn't checking for (according to https://coin.space/all-about-address-types/ 'ltc1' is another prefix option that this gem isn't looking for) 🧐

Reply:
Shouldn't that be added to the Litecoin validation then?